content/sigstore-feb24-roundup-2024.md

```diff
@@ -0,0 +1,116 @@
++++
+
+title = "Sigstore February Roundup"
+date = "2024-03-01"
+tags = ["sigstore","securesupplychain"]
+draft = false
+author = "Luke Hinds (TSC Chair)"
+type = "post"
+
++++
+
+Welcome to the February edition of the Sigstore Roundup! This is a regular summary of Sigstore news, events, releases and other happenings.
+
+### Events
+
+#### KubeCon Europe 2024
+
+The next KubeCon Europe will be held on **19th – 22nd March**.
+
+There are serveral Sigstore related talks and events planned for KubeCon Europe, including:
+
+- [Securing the Supply Chain with Sigstore Artifacts Signatures at Scale - Dmitry Savintsev & Yonghe Zhao, Yahoo](https://kccnceu2024.sched.com/event/1YeNE)
+- [Navigating the Software Supply Chain Defense Landscape - Marina Moore & Aditya Sirish A Yelgundhalli, New York University](https://kccnceu2024.sched.com/event/1YeO3/navigating-the-software-supply-chain-defense-landscape-marina-moore-aditya-sirish-a-yelgundhalli-new-york-university)
+- [Contribfest: Enable Additional Signing Mechanisms for TUF and in-toto: No Cryptography Skills Required](https://kccnceu2024.sched.com/event/1YkRE/contribfest-enable-additional-signing-mechanisms-for-tuf-and-in-toto-no-cryptography-skills-required)
+
+#### FOSDEM 2024
+
+The next FOSDEM will be held in Brussels, Belgium on the **3rd & 4th February 2024**
```

Comment:
since this is in the past, should you refer to it as "was held"?

Reply:
Oh crap, that should not be there.